internal/dnfjson/dnfjson.go

```diff
@@ -0,0 +1,200 @@
+package dnfjson
```

Comment:
I think this commit message needs an explanation for what the purpose if this new package is, and how it differs from `rpmmd`, and what the intended end-state is.